ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -14,10 +14,29 @@ import ReactiveCocoa
 class SignalSpec: QuickSpec {
 	override func spec() {
 		describe("init") {
-			pending("should run the generator immediately") {
-			}
-
-			pending("should keep signal alive if not terminated") {
+			let testError = NSError(domain: "SignalSpec", code: 1, userInfo: nil)
+			var testScheduler: TestScheduler!
+			
+			beforeEach {
+				testScheduler = TestScheduler()
+			}
+			
+			it("should run the generator immediately") {
+				var didRunGenerator = false
+				Signal<Int, NoError> { observer in
+					didRunGenerator = true
+					return nil
+				}
+				
+				expect(didRunGenerator).to(beTrue())
+			}
+
+			it("should keep signal alive if not terminated") {
+				let signal: Signal<Int, NoError> = Signal { observer in
+					return nil
+				}
+				
+				expect(signal).toNot(beNil())
```

Comment:
The `signal` property is strong, so the signal will be kept alive regardless. A better test would be to put the signal into a `weak var` and see if it disappears.